unicore/distribute/api/tests/test_repos.py

```diff
@@ -71,7 +74,10 @@ def test_collection_post_success(self):
             '/repos/%s.json' % (api_repo_name,))
         self.assertEqual(request.response.status_code, 301)
 
-    def test_collection_post_error(self):
+    @patch.object(EG, 'clone_repo')
+    def test_collection_post_error(self, mock_method):
+        mock_method.side_effect = GitCommandError(
+            'git clone', 'Boom!', stderr='mocked response')
```

Comment:
This change is because we actually tried cloning stuff from `example.org` which made stuff very slow. In a way unrelated to the feature being added in this PR.